statistics/handle/cache/statscacheinner.go

```diff
@@ -129,9 +164,15 @@ func (sc *StatsCache) Cost() int64 {
 
 // SetCapacity sets the memory capacity of the cache.
 func (sc *StatsCache) SetCapacity(c int64) {
+	metrics.CostGauge.Set(float64(c))
 	sc.c.SetCapacity(c)
 }
 
+// Stop stops the cache.
+func (sc *StatsCache) Stop() {
```

Comment:
Better to use `Close` to aligned with the underlying cache's method name.
And in addition, `Close` seems a better name in Golang than `Stop`, many standard libs use `Close` instead of `Stop` like `io.Closer`.

Reply:
Done